src/modules/CRUD/components/Nav/Nav.js

```diff
@@ -54,32 +42,35 @@ export const Nav = ({ getAllLayersAction, resizingMap, layersList, t }) => {
           minimal
         />
       </div>
-      <ul
-        id="CRUD-nav__menu"
-        className="CRUD-nav__menu"
-      >
-        {layersListByGroups.map(({ group, list }) => (
-          <li className="CRUD-nav__group" key={group}>
-            <p className="CRUD-nav__group-name">
-              {group}
-            </p>
-            <ul className="CRUD-nav__list">
-              {list.map(({ name }) => (
-                <li className="CRUD-nav__item" key={name}>
-                  <NavLink
-                    to={generateURI('layer', { layer: name })}
-                  >
-                    <span className="bp3-button bp3-minimal">
-                      <Icon icon="layout" />
-                      <span className="bp3-button-text">{name}</span>
-                    </span>
-                  </NavLink>
-                </li>
-              ))}
-            </ul>
-          </li>
-        ))}
-      </ul>
+      { menu && (
+        <ul
+          id="CRUD-nav__menu"
+          className="CRUD-nav__menu"
+        >
+          {menu.sort(sortByOrder).map(({ name: group, pictogram: groupPictogram, crud_views: list }) => (
```

Comment:
It's not a good idea to `.sort()` array on each component render...

You should store sorted arrays as state that is recompute only on `menu` prop change.
Maybe something like:

```js
const [orderedMenu, setOrderedMenu] = useState(menu);

useEffect(() => {
  setOrderedMenu(menu.sort(SortByOrder));
  // but you also have menu sub items (`crud_views`) to manage...

  // You could propably extract such a sorting function into an helper:
  setOrderedMenu(getProperlyOrderedMenu(menu));
}, [menu]);
```

*...then menu would be processed only when `menu` prop is modified, and not on each render.*

Reply:
ok, it's now fixed.
I added a filter to exclude a group of layer we don't need in the menu